sgl-kernel/benchmark/bench_moe_align_block_size.py

```diff
@@ -360,15 +381,15 @@ def benchmark(num_tokens, num_experts, topk, provider):
     parser.add_argument(
         "--num_experts",
         type=int,
-        default=256,
+        default=8,
         choices=[8, 16, 32, 64, 128, 256],
         help="Number of experts for benchmark",
     )
     parser.add_argument(
         "--topk",
         type=int,
-        default=8,
-        choices=[2, 4, 8],
+        default=1,
```

Comment:
(same as above)

Reply:
I'll revert it too.

Reply:
(ignore this - dnk why my comment is repeated twice for L391...)